tests/unit/s2n_safety_test.c

```diff
@@ -286,6 +286,8 @@ int main(int argc, char **argv)
     uint8_t e[4] = { 1, 2, 3, 4 };
     uint8_t f[4] = { 1, 2, 3, 5 };
 
+    uint8_t all_zero[4] = { 0, 0, 0, 0 };
```

Comment:
nit: we normally try and follow a more modern convention where variables are declared before they are used rather than being declared at the top of the function.

So I'd vote for moving all of the lines of this new test (both declaration and the actual check) to line 281ish so that it isn't in the middle of the a/b/c/ test.

Reply:
Done.